future-apps/payroll/contracts/Payroll.sol

```diff
@@ -476,17 +475,17 @@ contract Payroll is AragonApp { //, IForwarder { // makes coverage crash (remove
         return allowedTokens[_token];
     }
 
-    /**
-     * @dev IForwarder interface conformance. Forwards any employee action.
-     * @param _evmScript script being executed
-     */
-    function forward(bytes _evmScript) public {
-        require(canForward(msg.sender, _evmScript));
-        bytes memory input = new bytes(0); // TODO: Consider input for this
-        address[] memory blacklist = new address[](1);
-        blacklist[0] = address(finance);
-        runScript(_evmScript, input, blacklist);
-    }
+    // /**
+    //  * @dev IForwarder interface conformance. Forwards any employee action.
+    //  * @param _evmScript script being executed
+    //  */
+    // function forward(bytes _evmScript) public {
+    //     require(canForward(msg.sender, _evmScript));
+    //     bytes memory input = new bytes(0); // TODO: Consider input for this
+    //     address[] memory blacklist = new address[](1);
+    //     blacklist[0] = address(finance);
+    //     runScript(_evmScript, input, blacklist);
+    // }
```

Comment:
Why remove this function?

Reply:
@Quazia  We were having a `Out of gas` error when deploying the contract and it seemed to be due to byte code size so we commented the function just for workaround that problem. It is back in this other PR [#19](https://github.com/protofire/aragon-apps/pull/19)